mode/examples/Topics/Image Processing/Blending/Blending.pyde

```diff
@@ -18,17 +18,18 @@ modes = ((REPLACE, "REPLACE"),
          (MULTIPLY, "MULTIPLY"),
          (SCREEN, "SCREEN"),
          (REPLACE, "REPLACE"))
-img1 = loadImage("layer1.jpg")
-img2 = loadImage("layer2.jpg")
+
 currentMode = 0
 
 def setup():
-    size(640, 360, P3D)
+    size(640, 360, P3D);
     noStroke()
 
 
 def draw():
     global picAlpha
+    img1 = loadImage("layer1.jpg")
```

Comment:
No; you mustn't load the image 60 times per second. You should do it in `setup()`.